__tests__/app.test.js

```diff
@@ -76,4 +76,32 @@ describe("Nc News, testing API's", () => {
         });
     });
   });
+  describe.only("GET, /api/users", () => {
+    it("should respond with an array of user objects ", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((res) => {
+          const users = res.body.users;
+          expect(users).toHaveLength(4);
+          users.forEach((user) => {
+            expect(user).toEqual(
+              expect.objectContaining({
+                username: expect.any(String),
+                name: expect.any(String),
+                avatar_url: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+    it("should respond with 404 - Not found when path is incorrect", () => {
+      request(app)
+        .get("/api/isers")
+        .expect(404)
```

Comment:
This should be a universal error test for any incorrect endpoint, not just for /api/users. You can put it in its own describe block

Reply:
Hi Rayhaan, thanks for the feedback! I've changed it now, is it good to merge?